### PR TITLE
Format RDS-S3 Readme for readability

### PR DESCRIPTION
**Pull request by mbaijal**
**Description of your changes:**
Format RDS-S3 ReadMe for readability

### CODE REVIEW

**Review by AlexandreBrown — APPROVED** [review]

File: distributions/aws/examples/rds-s3/README.md [hunk 1]
```diff
@@ -77,12 +77,12 @@ Replace `YOUR_CLUSTER_REGION`, `YOUR_CLUSTER_NAME` and `YOUR_S3_BUCKET` with you
...
-##### **Advanced customization**
+### Advanced customization
 The script applies some sensible default values for the db user password, max storage, storage type, instance type etc but if you know what you are doing, you can always tweak those preferences by passing different values.  
 You can learn more about the different parameters by running `python auto-rds-s3-setup.py --help`.  
 
 
-#### **Manual setup**
+### 2.2 **Option 2: Manual Setup**
 If you prefer to manually setup each components then you can follow this manual guide.  
 1. Create S3 Bucket
```
Comment (akartsky) [inline distributions/aws/examples/rds-s3/README.md:87]: shouldn't this be 2.2.1 ?

File: distributions/aws/examples/rds-s3/README.md [hunk 2]
```diff
@@ -59,8 +57,10 @@ eksctl create cluster \
 **Important :**  
 You must make sure you have an [OIDC provider](https://docs.aws.amazon.com/eks/latest/userguide/enable-iam-roles-for-service-accounts.html) for your cluster and that it was added from `eksctl` >= `0.56` or if you already have an OIDC provider in place, then you must make sure you have the tag `alpha.eksctl.io/cluster-name` with the cluster name as its value. If you don't have the tag, you can add it via the AWS Console by navigating to IAM->Identity providers->Your OIDC->Tags.
 
-### 2. Setup RDS and S3
-#### **Automated setup**
+## 2.0 Setup RDS, S3 and configure Secrets
```
Comment (AlexandreBrown) [inline distributions/aws/examples/rds-s3/README.md:60]: Really appreciate the changes here, much more readable.  
The only thing I'd say is check to make sure `cognito-rds-s3` README link is updated for this new H title name.  
Check the 1. Prerequisite par `RDS & S3 from [this guide](https://github.com/mbaijal/kubeflow-manifests/blob/fix-rds-readme/distributions/aws/examples/rds-s3/README.md#2-setup-rds-and-s3) to:`

Comment (AlexandreBrown) [inline distributions/aws/examples/rds-s3/README.md:60]: Really appreciate the changes here, much more readable.  

The only thing I'd say is check to make sure `cognito-rds-s3` README link is updated for this new H title name.  
Check the 1. Prerequisite par `RDS & S3 from [this guide](https://github.com/mbaijal/kubeflow-manifests/blob/fix-rds-readme/distributions/aws/examples/rds-s3/README.md#2-setup-rds-and-s3) to:`

File: distributions/aws/examples/rds-s3/README.md (hunk 1 above)
Comment (mbaijal) [reply distributions/aws/examples/rds-s3/README.md:87]: I think that gets messy. Also I checked the other READMEs and it's kinda this way. I am ok to change though if we want.

File: distributions/aws/examples/rds-s3/README.md (hunk 2 above)
Comment (mbaijal) [reply distributions/aws/examples/rds-s3/README.md:60]: Good point !

**Review by AlexandreBrown — APPROVED** [review]

File: distributions/aws/examples/rds-s3/README.md [hunk 3]
```diff
@@ -183,19 +183,20 @@ Resources:
...
 while ! kustomize build distributions/aws/examples/rds-s3 | kubectl apply -f -; do echo "Retrying to apply resources"; sleep 10; done
 ```
 
-Once, everything is installed successfully, you can access the Kubeflow Central Dashboard [by logging in to your cluster](../../../../README.md#connect-to-your-kubeflow-cluster).
+After everything is installed successfully, you can access the Kubeflow Central Dashboard [by logging in to your cluster](../../../../README.md#connect-to-your-kubeflow-cluster).
 
 Congratulations! You can now start experimenting and running your end-to-end ML workflows with Kubeflow.
 
-## Verify the instalation
+## 4.0 Verify the instalation
```
Comment (surajkota) [inline distributions/aws/examples/rds-s3/README.md:197]: typo: installation

File: distributions/aws/examples/rds-s3/README.md (hunk 2 above)
Comment (mbaijal) [reply distributions/aws/examples/rds-s3/README.md:60]: fixed.

File: distributions/aws/examples/rds-s3/README.md (hunk 3 above)
Comment (mbaijal) [reply distributions/aws/examples/rds-s3/README.md]: fixed.

**Review by surajkota — APPROVED** [review]

**Pull request merged**